src/microbe_stage/PlayerMicrobeInput.cs

```diff
@@ -66,7 +66,7 @@ public override void _Input(InputEvent @event)
 
         if (@event.IsActionPressed("g_cheat_editor"))
         {
-            // TODO: reimplement editor enter cheat
+            stage.HUD.EditorButtonPressed();
```

Comment:
Actually, the cheat used to just enable the editor button, not directly go to the editor. I think I'd like to get that functionality back.